build/Build.AzurePipelinesAttribute.cs

```diff
@@ -15,8 +15,11 @@ partial class Build
 {
     public class AzurePipelinesAttribute : Nuke.Common.CI.AzurePipelines.AzurePipelinesAttribute
     {
-        public AzurePipelinesAttribute(AzurePipelinesImage image, params AzurePipelinesImage[] images)
-            : base(image, images)
+        public AzurePipelinesAttribute(
+            string suffix,
```

Comment:
this looks a breaking change but it was released in PATCH version?

Reply:
Sorry, but until we hit 1.0 there is no guarantee for breaking changes. However, I'm trying to keep them as small as possible, and you also can always ask in our [Slack workspace](https://slofile.com/slack/nukebuildnet).

Reply:
@ryancyq from next hotfix release there will be a new constructor allowing to not pass a suffix.. Sorry I forgot thinking of a second constructor to make it optional.

https://github.com/nuke-build/nuke/pull/456

Reply:
Cool, thanks for the update 👍🏻